Buildings/ThermalZones/Detailed/Examples/ISAT/ForcedConvection.mo

```diff
@@ -0,0 +1,272 @@
+within Buildings.ThermalZones.Detailed.Examples.ISAT;
+model ForcedConvection
+  "Ventilation with forced convection in an empty room"
+  extends Modelica.Icons.Example;
+  extends Buildings.ThermalZones.Detailed.Examples.ISAT.BaseClasses.PartialRoom(
+    roo(
+      surBou(
+        name={"East Wall","West Wall","North Wall","South Wall","Ceiling",
+            "Floor"},
+        A={0.9,0.9,1,1,1,1},
+        til={Buildings.Types.Tilt.Wall,Buildings.Types.Tilt.Wall,Buildings.Types.Tilt.Wall,
+            Buildings.Types.Tilt.Wall,Buildings.Types.Tilt.Ceiling,Buildings.Types.Tilt.Floor},
+        each absIR=1e-5,
+        each absSol=1e-5,
+        each boundaryCondition=Buildings.ThermalZones.Detailed.Types.CFDBoundaryConditions.Temperature),
+      nPorts=2,
+      portName={"Inlet","Outlet"},
+      cfdFilNam=
+          "modelica://Buildings/Resources/Data/ThermalZones/Detailed/Examples/ISAT/ForcedConvection/input.ffd",
+      samplePeriod=200,
+      linearizeRadiation=true,
+      massDynamics=Modelica.Fluid.Types.Dynamics.DynamicFreeInitial),
+      nSurBou=6);
+
+  Buildings.Fluid.Sources.Boundary_pT bouOut(
+    nPorts=1,
+    redeclare package Medium = MediumA)
```

Comment:
ensure there are comments to all instances.

Reply:
Revised.